oic.r.acl2.raml

```diff
@@ -177,97 +204,28 @@ traits:
-                    "rel": "some-rel",
-                    "rt": ["oic.r.light"],
-                    "if": ["oic.if.a"]
-                  },
-                  {
-                    "href": "coaps://IP-ADDR/door",
-                    "rel": "some-rel",
-                    "rt": ["oic.r.door"],
-                    "if": ["oic.if.a"]
-                  }
-                ],
-                "permission": 24,
-                "validity": [
-                  {
-                    "period":"20160101T180000Z/20170102T070000Z",
-                    "recurrence": [ "DSTART:XXXXX", "RRULE:FREQ=DAILY;UNTIL=20180131T140000Z;BYMONTH=1" ]
-                  },
-                  {
-                    "period":"20160101T180000Z/PT5H30M",
-                    "recurrence": [ "RRULE:FREQ=DAILY;UNTIL=20180131T140000Z;BYMONTH=1" ]
-                  }
-                ]
-              },
-              {
-                "subject": {
-                  "uuid": "e61c3e6b-9c54-4b81-8ce5-f9039c1d04d9"
-                },
-                "resources": [
-                  {
-                    "href": "coaps://IP-ADDR/light",
-                    "rel": "some-rel",
-                    "rt": ["oic.r.light"],
-                    "if": ["oic.if.a"]
-                  },
-                  {
-                    "href": "coaps://IP-ADDR/door",
-                    "rel": "some-rel",
-                    "rt": ["oic.r.door"],
-                    "if": ["oic.if.a"]
-                  }
-                ],
-                "permission": 24
-              }
-            ],
-            "rowner": {
-              "hostid": {
-                "uuid": "e61c3e6b-9c54-4b81-8ce5-f9039c1d04d9"
-              }
-            }
-          }
-
-    responses:
-      201:
-        description: The ACL entry is created.
-      400:
-        description: The request is invalid.
-
   delete:
+
     description: |
       Deletes ACL entries.
-      When DELETE is used without query parameters, all the acl entries are deleted.
-      When DELETE is used with a subjectuuid query parameter, only the entries matching the subjectuuid are deleted.
+      When DELETE is used without query parameters, all the ACE entries are deleted.
+      When DELETE is used with an aceid or resources query parameter, only the
+      ACEs matching the specified parameter are deleted.
     queryParameters:
-        subjectuuid:
-            type: string
-            description: |
-              Deletes the ACL(s) identified by the string containing subjectuuid.
-              e.g. DELETE /myacl?subjectuuid="de305d54-75b4-431b-adb2-eb6b9e546014"
-            required: false
-            example: |
-                DELETE /myacl?subjectuuid="de305d54-75b4-431b-adb2-eb6b9e546014"
-
+      aceid:
+          type: integer
+          description: |
+            Deletes the ACE with the specificed aceid
+          required: false
+          repeat: false
+          example: 2112
+      resources:
```

Comment:
Looking closer at the IoTivity code, "resources" is only processed in conjunction with a subject. So do we want to retain that pattern or does "resources" make sense by itself?